README.md

```diff
@@ -1,6 +1,6 @@
 # Getting Started with Create React App
```

Comment:
- [ ] Kindly update your README file to contain a proper description of the project.